### ISSUE TITLE
Use less scary Minerva login fail message

### ISSUE THREAD

**Issue by wschella** [ZeusWPI/hydra-android#166]
When the Minerva API token expires, users are alerted with this scary "Error using Minerva credentials. Please relog". Or something close to that, the important part is *Error*. It isn't really an error when we know it's going to happen. We differentiate  when the error is caused by an expired token, and reflect that to the user. Something the likes of: "Login expired. Renew to keep receiving Minerva updates".

**Comment by niknetniko** [comment]
Since 2.4 the Android framework handles showing that notification. To change what is displayed, we need to disable Android's handling of the error and do it ourselves.

**Comment by wschella** [comment]
From user feedback, this seems like a really big annoyance. I kinda have to agree, and propose to raise priority. 

The simplest alternative i could come up with is a one tjme only message saying: "Login expired. No more updates until you relog"

**Comment by niknetniko** [comment]
While I agree the notification is annoying, it should only appear every ~1-2 months or so (from my personal experience). At least that what I think, I don't know what the actual expire date is.

If we use a one-time notification, I wouldn't be surprised if the user dismisses the notification, not (or only partially) reading it, thinking "I'll handle this later". Out of sight is out of mind, and then the user will probably blame the app with things like "Why does it not sync my announcements?".

We could show the notification once and then display a message somewhere in the app itself if the user visits the Minerva portion of the app. But this would require some substantial changes, and as such can't be done quickly.

Another aspect is that the notification isn't _wrong_. When the tokens expire, the Minerva synchronisation stops working. If the user relies on the app to receive notifications, e.g. only checking the desktop version once in a while and not checking emails, it is probably important that the synchronisations keep happening.

**TL;DR:**
- Yes, they are annoying, but should only occur every ~1-2 months.
- On one hand, we don't want to annoy the user with notifications. On the other hand, the user expects and might rely on the synchronisation to work.

**Real solution**
The people from Minerva should increase the lifespan of their refresh tokens. Expiring refresh tokens is why the notification appears in the first place. I assume it's their code that is expiring, but to be sure, I'll do another review of the OAuth code in the app (see also #176) to make sure the app doesn't do anything wrong somewhere that prevents the tokens from being re-used.

**Comment by wschella** [comment]
Just a quick comment, it happens only 1-2 months, if you log in immediately, but it keeps bugging you otherwise, even in vacations. Could we just put an exclamation mark on the sync buttons when you are not logged in?

**Comment by niknetniko** [comment]
#295 removed the Minerva integration, meaning this no longer applies.
